src/main/java/eu/dissco/core/translator/terms/License.java

```diff
@@ -10,28 +9,24 @@ public class License extends Term {
 
   public static final String TERM = "dcterms:license";
   private final List<String> dwcaTerms = List.of(TERM, "dc:license");
+  private final List<String> abcdUnitTerms = List.of("abcd:iprstatements/licenses/license/0/uri",
+      "abcd:iprstatements/licenses/license/0/text");
+  private final List<String> abcdMetaTerms = List.of(
+      "abcd:metadata/iprstatements/licenses/license/0/uri",
+      "abcd:metadata/iprstatements/licenses/license/0/text");
```

Comment:
Can you combine these lists? Or do you want to make sure Unit terms take precedence over meta terms?

Reply:
Yes, the path is different and the object is searches over is also different, so it is hard to combine these two lists.